app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -110,6 +115,10 @@ class FxaRepo(
         return accountManager.beginAuthenticationAsync()
     }
 
+    fun lastTabCouldNotBeDisplayed() {
```

Comment:
Yeah, I think you have a very fair critique here.  If we were being strict, this would all go through a VM that converted the `something happened` into a `do this thing`.  As it is, I'll take your suggestion.